cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/DatabaseBrowser/index.js

```diff
@@ -143,6 +143,48 @@ export default class DatabaseBrowser extends Component {
       );
   }
 
+  renderEmpty(tables) {
+    if (tables.length === 0 && this.state.search.length !== 0) {
```

Comment:
The `tables.length === 0` check can be removed, since outside of this we're only calling `renderEmpty` when `!tables.length`. Also, this conditional block of code would be shorter if we do `if (this.state.search.length === 0)`